tvb_contrib/tvb/contrib/simulator/models/epileptor.py

```diff
@@ -154,28 +156,16 @@ class HMJEpileptor(models.Model):
         default=("y0", "y3"),
         doc="""default state variables to be monitored""")
 
-#    variables_of_interest = arrays.IntegerArray(
-#        label="Variables watched by Monitors",
-#        range=basic.Range(lo=0.0, hi=6.0, step=1.0),
-#        default=numpy.array([0], dtype=numpy.int32),
-#        doc="default state variables to be monitored",
-#        order=10)
-
-
     def __init__(self, **kwargs):
         """
         """
-
-        LOG.info("%s: init'ing..." % (str(self),))
-
         super(HMJEpileptor, self).__init__(**kwargs)
```

Comment:
Looks like we could take out the `__init__` from contrib models to make them consistent with the library models, the attributes set inside `__init__` could move directly on the class.

Reply:
Is it okay now?